### PR TITLE
plugins: enable needs-rebase for openstack repos

### PR DESCRIPTION
**Pull request by dims**
Change-Id: Ie61b35fdba586fe4d93fc7c7b3419599a15e906d

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -724,6 +728,10 @@ external_plugins:
     events:
       - pull_request
   kubernetes-sigs/cluster-api-provider-digitalocean:
```
Comment (BenTheElder) [inline prow/plugins.yaml]: Are this and k/contrib openstack repos?

Comment (dims) [reply prow/plugins.yaml]: the diff is a bit funny! (unless i messed something up). i need the plugin enabled for `kubernetes/cloud-provider-openstack` and `kubernetes-sigs/cluster-api-provider-openstack`

Comment (BenTheElder) [reply prow/plugins.yaml]: oh I see it now! heh

**Pull request merged**